### PR TITLE
chore: remove DB based leases

### PR DESCRIPTION
**Pull request by stuartwdouglas**
This means async calls no longer have linked leases, but as they are going away anyway that should not be a problem.

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
This seems to work now the hermit stuff is fixed, the helm charts need to be updated though, and possibly some kube tests.

File: charts/ftl/templates/lease-services.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  labels:
+    {{- include "ftl.labels" . | nindent 4 }}
+  name: {{ include "ftl.fullname" . }}-lease
+  {{- if .Values.lease.service.annotations }}
+  annotations:
+    {{- toYaml .Values.service.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  ports:
+    {{- range .Values.lease.ports }}
```
Comment (worstell) [inline charts/ftl/templates/lease-services.yaml:13]: should this be `{{- range .Values.lease.service.ports }}`? and should it be `{{- toYaml .Values.lease.service.annotations | nindent 4 }}` above?

File: charts/ftl/templates/lease-services.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    - name: {{ .name }}
+      port: {{ .port }}
+      protocol: {{ .protocol | default "TCP" }}
+      targetPort: {{ .targetPort }}
+    {{- end }}
+  selector:
+    {{- include "ftl-lease.selectorLabels" . | nindent 4 }}
+  type: {{ .Values.controller.leaseService.type | default "ClusterIP" }}
```
Comment (worstell) [inline charts/ftl/templates/lease-services.yaml:21]:
```suggestion
  type: {{ .Values.lease.service.type | default "ClusterIP" }}
```

**Review by worstell — APPROVED** [review]

File: charts/ftl/templates/lease-services.yaml (hunk 1 above)
Comment (stuartwdouglas) [reply charts/ftl/templates/lease-services.yaml]: yes, it should

**Pull request merged**